app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,5 @@
+class RegistrationsController < ApplicationController
+  def new
+    
```

Comment:
remove this empty line, the result could be
```rb
def new
end
```

or you can use this convention for controller methods that only render a view with the same name

```rb
def new; end
```